### DISCUSSION TITLE
django and rqlite

### DISCUSSION THREAD

**Discussion by madsholme** [rqlite/rqlite#1919] [General]
I was in the process to move my app from sqllite to rqlite, but im the process saw the django driver, what i need for some api stuff does not work (its 6 years old so properly abandoned now).. are there any plans or any way to get rqlite working in django? i dont mind writing raw sql, the orm does not need to work.

**Comment by otoolep** [comment]
No, I do not have the bandwidth to do anything like that. Best I can suggest is to check out the client libraries.

https://rqlite.io/docs/api/client-libraries/

**Comment by otoolep** [comment]
To be clear, I would welcome Django support for rqlite, but it's probably a bunch of work, and rqlite doesn't support traditional transactions so some things might be difficult to do via Django (I have built Django websites in the past so know a little bit about this).

So if folks are interested in taking stab at it, great, just not sure how feasible it is.

**Reply by madsholme** [reply]
thanks for the answer.. I was hoping there at least was an easy way to make select and simple incert into statements through pyrqlite, even without orm support. But i guess i can do some hacky stuff, and just use the pyrqlite and transforming the results into json objects manualy....

but its properly verry niche so i can understand why its not ben done.. would be nice though...
